### DISCUSSION TITLE
有关数据集合增加维度与完成归档功能等建议

### DISCUSSION THREAD

**Discussion by Nemo137-Liang** [shion-app/shion#53] [Ideas]
目前数据集合中有“计划”与“计划/标签”维度，以及当特定事项完成时没有归档功能。

**1. 希望在“计划”与“标签”之外，增加第3个维度。** 

例如：我添加一项“阅读”的计划，并在该计划下添加“《好兆头》”与“《世界通史》”这2个标签，希望可以有第3个维度，将“《好兆头》”标记为“小说”、“《世界通史》”标记为”专著“，这样在数据的展示时，也可以展示为”阅读/小说“、”阅读/专著”。或者增加一个类似“摘要“或”备注“的维度，我可以在阅读的计划下，添加小说、专著等标签，再使用摘要或备注写明具体的书，同时将相同摘要的时间记录进行合并。

又或者，我可以给计划打上标记，计划是《女神异闻录5》，计划下的标签分为“一周目”、“二周目”，然后将这个计划标记为“游戏”这个大分类。而“游戏”这个大分类下，我还可以添加《WOW》为计划，并在这个计划下添加“主线”、“团本开荒”这不同的标签。

**2. 希望增加完成或归档功能**

希望第2级的数据（在现版本下为“标签”）增加“完成”功能。例如我在看完一本书后，那么将该标签设置为“完成”，后续就无法再对其进行计时。

**Comment by hanaTsuk1** [comment]
> 希望在“计划”与“标签”之外，增加第3个维度。

这个我之前也想过类似的功能，但一直没决定下来。我觉得有必要但是单独加上来，又觉得功能太单薄
我最近在弄obsidian，不知道你用过没有，虽然是个文本编辑应用，但是相关的生态和插件功能让我觉得能和这个应用结合到一起
所以你说的这个我会考虑，等我有新的想法了会在这留言

> 希望增加完成或归档功能

说实话，我感觉意义不大

**Reply by Nemo137-Liang** [reply]
Obsidian和Logseq我都是在使用的，其实我很喜欢它们用双链将一个一个记录串联起来的基础设计。总之感谢开发者的回复，也期待Shion会更好。
